node/building_blocks.re

```diff
@@ -58,6 +58,53 @@ let is_current_producer = (state, ~key) => {
 // TODO: bad naming
 // TODO: check if block must have published a new snapshot
 let is_signable = (state, block) => {
+  open Protocol.Operation.Side_chain;
+  let Node.{
+        trusted_validator_membership_change,
+        protocol: {last_seen_membership_change_timestamp, _},
+        _,
+      } = state;
+  let allowed_op_reception_delay = 0.200; // Assumed max network latency
```

Comment:
The max network latency is closer to a second

Reply:
Addressed by getting rid of it - timestamps are no longer a part of the operation design now.